searx/templates/oscar/preferences.html

```diff
@@ -50,9 +55,11 @@ <h5><a href="{{about.website}}" rel="noreferrer">{{about.website}}</a></h5>
 {% endif %}
 {% if checker_result or errors %}
 <td class="{{ css_align_class }} {{ label }}">{{- "" -}}
-    <span aria-labelledby="{{engine_name}}_reliablity">
-        {%- if reliabilities[engine_name].checker %}{{ icon('exclamation-sign', 'The checker fails on the some tests') }}{% endif %} {{ r -}}
-    </span>{{- "" -}}
+    <a href="{{ url_for('stats', engine=engine_name|e) }}">{{- "" -}}
+        <span aria-labelledby="{{engine_name}}_reliablity">
+            {%- if reliabilities[engine_name].checker %}{{ icon('exclamation-sign', 'The checker fails on the some tests') }}{% endif %} {{ r -}}</a>
```

Comment:
Should the condition for the exclamation-sign based on_ checker_ or on _errors_?

    {%- if reliabilities[engine_name].checker %}

versus ..

    {%- if reliabilities[engine_name].errors %}

Compare: In 0bd35f799 I used `reliabilities[<engine.name>].errors`

Reply:
Following what I have understood of the ARIA recommendation: yes, if should be `{%- if reliabilities[engine_name].errors %}`

---
The issue is how to display the checker result without displaying a spreadsheet.

Here some context:

`r` ( `= reliabilities.get(engine_name, {}).get('reliablity')` ) is the percentage of requests that have not raised an exception ; but even the requests don't raise exceptions, the checker can see the engine as broken: 
* [supported language](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/search/processors/online.py#L193-L201), [time range](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/search/processors/online.py#L185-L191), [safe search](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/search/processors/online.py#L203-L208) results are [reported in the `/preferences` page](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/templates/oscar/macros.html#L138-L139). It is "strange" to report 100% reliability and at the same time to report some broken features.
* if the `simple` test fails,  `r` is set 0:
https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/webapp.py#L943-L946
* if [paging](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/search/processors/online.py#L174-L183) or [rosebud](https://github.com/searxng/searxng/blob/fe3831db1aa73ebda6eb12fa3a321002c176ccc1/searx/settings.yml#L123-L131) tests fail, there are reports other than:
   * this exclamation mark next to the `r` value.
   * the tooltip on the `r` value.

Reply:
What about
`{%- if reliabilities[engine_name].errors or reliabilities[engine_name].checker %}`
?

Reply:
> What about

LGTM 👍🏻

Reply:
> of the ARIA recommendation: 

shame on me, did not know about ARIA .. 🤦🏻  .. I think I have to read some paper / now I see the usage of `aria-labelledby`.

> So if the exclamation mark is displayed with this condition {%- if reliabilities[engine_name].errors %}, how to display that the paging test fail ?

If I have understand everything, I think your suggestion in https://github.com/searxng/searxng/pull/40#discussion_r622788834 is the best solution.  May be instead of the text 

    'The checker fails on the some tests'

we should choose a text which is more general / e.g.: _The engine fails for some reason._

Reply:
I'm going to open another for this (and the typo `{{engine_name}}_reliablity` --> `{{engine_name}}_relialibility`

Reply:
> I'm going to open another for this

fine .. feel free to merge this PR, if you think it is OK / for me it is OK.